### ISSUE TITLE
Template with comment on first line are not work

### ISSUE THREAD

**Issue by brutalcrozt** [formly-js/angular-formly#644]
I just hitting this issue, and not explicitly request feature.
Feel free to improve or just make an explicit warning on the documentation.
[Isolated issue](http://jsbin.com/gaxoqo/edit?html,js,output)

**Comment by BarryThePenguin** [comment]
It looks like this is an issue with [formly-js/angular-formly-templates-bootstrap](https://github.com/formly-js/angular-formly-templates-bootstrap/blob/bf39c6fc1eac9181c269c45861164140dedb91bd/src/run/description.js)

Specifically where the first item from the template is appended to the description manipulator.

```js
var el = document.createElement('div');
el.appendChild(angular.element(template)[0]);
```

**Comment by brutalcrozt** [comment]
yeah thanks for pointing that, i may take a look later.
too much deadline :cry:

**Comment by ajay2507** [comment]
Is this still open?. Can I take this issue?

**Comment by brutalcrozt** [comment]
yes @ajay2507 , we will be happy if you want take this down :) .
FYI we are building the formly for angular 2 at [ng-formly](https://github.com/formly-js/ng-formly). 
You may submit pr for v.1 or v.2 or both. But i am not sure if the version 2 is aware of this.
